server/database/handlers/notificationDBHandler.js

```diff
@@ -0,0 +1,77 @@
+const models = require("../models");
+const Notification = models.Notification;
+const Product = models.Products;
+const CronJob = require("cron").CronJob;
+const { scrapeAmazon, scrapeEbay } = require("../../scrapers/index");
+
+async function createNotification(
+  productId,
+  userId,
+  previousPrice,
+  currentPrice
+) {
+  const notification = await Notification.create({
+    productId: productId,
+    userId: userId,
+    previousPrice: previousPrice,
+    currentPrice: currentPrice,
+  })
+    .then((res) => {
+      return res;
+    })
+    .catch((err) => {
+      console.log(err);
+      console.log("Notification not created.");
+      return null;
+    });
+  return notification;
+}
+
+const getSalePrices = async function () {
+  const notifications = [];
+  await Product.findAll().then((products) => {
```

Comment:
Quick Question: so did this code actually work for you? Because when I was making the queries, I kept getting an error of a extra column that I never actually instantiated. So I had to keep declaring which attributes I wanted returned, just so I wouldn't get the error.

i.e. I had to type `Products.findAll({attributes: ["productId", "productName", "productImageURL", "productSalePrice"]})`

If it's just me, then maybe I installed sequelize/postgres wrong or something along those lines.

Reply:
Also, isn't the table called *`Products` rather than `Product`?